dapps/satoshi-multiplayer/contract/sources/tournament.move

```diff
@@ -59,7 +72,15 @@ module contract::tournament {
 
     // Match error codes.
     const EMatchNotFound: u64 = 7;
-    const EMatchStillRunning: u64 = 11;
+    const EMatchStillRunning: u64 = 12;
+
+    // Transfers admin capability to address deploying the contract.
+    fun init(ctx: &mut TxContext){
+        let id = object::new(ctx);
+        let admin_address = tx_context::sender(ctx);
+        let adminCap = AdminCap{ id , admin_address};
+        transfer::transfer(adminCap, tx_context::sender(ctx));
```

Comment:
DRY (do not repeat yourself) use admin_address instead of tx_context::sender(ctx) everywhere if you will bind this to a variable (in the last transfer)